FlightStreamDeck.SimConnectFSX/SimConnectFlightConnector.cs

```diff
@@ -715,6 +714,21 @@ public void RegisterSimValue(TOGGLE_VALUE simValue)
             }
         }
 
+        public void RegisterSimValue(TOGGLE_VALUE simValue, ValueEntry simValueEntry)
+        {
+            var changed = false;
+            lock (lockLists)
+            {
+                logger.LogInformation("Registering {key} {value}", simValue, simValueEntry);
+                changed = genericValues.Add(simValue);
```

Comment:
Do you plan to allow multiple buttons with different unit on the same variables in this PR? 
This line would lead to the unit of the first button be used for all others.